source/includes/code-examples/find-many/FindMany.cs

```diff
@@ -12,22 +14,22 @@ public static void Main(string[] args)
     {
         Setup();
 
-        // Find multiple documents using builders
+        // Finds multiple documents using builders
         Console.WriteLine("Finding documents with builders...:");
         var restaurants = FindMultipleRestaurantsBuilderSync();
         Console.WriteLine($"Number of documents found: {restaurants.Count}");
 
-        // Extra space for console readability 
+        // Prints extra space for console readability 
         Console.WriteLine();
 
-        // Find multiple documents using LINQ
+        // Finds multiple documents using LINQ
```

Comment:
Should I leave in comments that call the methods that actually do the work?

Reply:
Yeah I think this comment mentioning LINQ is good, I'd just change "using" to "by using"